mldsa/fips202/fips202.c

```diff
@@ -562,6 +562,45 @@ __contract__(
   s[(r - 1) / 8] ^= 1ULL << 63;
 }
 
+/*************************************************
+ * Name:        keccak_squeezeblocks
+ *
+ * Description:
+ *
+ * Arguments:   - uint8_t *out: pointer to output blocks
+ *              - size_t nblocks: number of blocks to be squeezed (written to
+ *out)
+ *              - uint64_t *s: pointer to input/output Keccak state
+ *              - unsigned int r: rate in bytes (e.g., 168 for SHAKE128)
+ **************************************************/
+static void keccakf1600_extract_bytes(uint64_t *state, unsigned char *data,
+                                      unsigned offset, unsigned length)
+__contract__(
+requires(0 <= offset && offset <= MLD_KECCAK_LANES * sizeof(uint64_t) &&
+    0 <= length && length <= MLD_KECCAK_LANES * sizeof(uint64_t) - offset)
+requires(memory_no_alias(state, sizeof(uint64_t) * MLD_KECCAK_LANES))
+requires(memory_no_alias(data, length))
+assigns(memory_slice(data, length)))
+{
+  unsigned i;
+#if defined(MLK_SYS_LITTLE_ENDIAN)
```

Comment:
`MLK_SYS_LITTLE_ENDIAN` I'm guessing from the prefix that this is an mlkem-native definition that doesn't exist in mldsa-native.

Reply:
It's `MLD_SYS_LITTLE_ENDIAN` in mldsa-native

Reply:
Sorry, I got lost between multiple github branches! I just updated it.